pkg/doc/verifiable/credential.go

```diff
@@ -374,6 +375,13 @@ func WithJWTPublicKeyFetcher(fetcher PublicKeyFetcher) CredentialOpt {
 	}
 }
 
+// WithUnsecuredJWT indicates the need to decode Verifiable Credential from unsecured JWT
```

Comment:
JWT or JWS?

(https://github.com/hyperledger/aries-framework-go/pull/425#discussion_r332511429)

Reply:
I'm using JWT (JSON Web Token) term here. And JWS for a signed one. Here we have neither signing nor encryption (JWE), so it's JWT.

Reply:
> Unsecured JWS
>       A JWS that provides no integrity protection.  Unsecured JWSs use
>       the "alg" value "none".
https://tools.ietf.org/html/rfc7515

Are you setting "alg: none"?

Reply:
I think we need to use the term "Unsecured JWT" solely.

Reply:
Should also update the commit title to remove "(plain)".

Reply:
Yes, thanks for the reminder!